eclair-core/src/main/scala/fr/acinq/eclair/io/Peer.scala

```diff
@@ -544,6 +553,8 @@ object Peer {
 
   val IGNORE_NETWORK_ANNOUNCEMENTS_PERIOD = 5 minutes
 
+  val MAX_RECONNECT_INTERVAL = 1 hour
```

Comment:
MAX_RECONNECT_INTERVAL should also be a configuration setting

Reply:
ACK for having this in the configuration settings.

Reply:
done with b8a056c